### PR TITLE
Updated sharepoint docs

### PR DESCRIPTION
**Pull request by alexandruatomei9**
Added an example of how a query should be built for the query operations

### CODE REVIEW

File: modules/ROOT/pages/sharepoint/sharepoint-connector.adoc [hunk 1]
```diff
@@ -196,6 +196,18 @@ For Files and Folders, operations include add, retrieve content, retrieve metada
 * What Mule editions can I use this connector with?
 +
 This connector is supported on any Enterprise Edition Anypoint platform running on any operating system and bit type, including the CloudHub integration PaaS.
++
+* How do I build a query for a query operation?
++
+The query operations `Folder Query`, `File Query` and `List Item Query` only support OData queries.
```
Comment (dhirsch1) [inline modules/ROOT/pages/sharepoint/sharepoint-connector.adoc]: Add a comma after `File Query`.

File: modules/ROOT/pages/sharepoint/sharepoint-connector.adoc [hunk 2]
```diff
@@ -196,6 +196,18 @@ For Files and Folders, operations include add, retrieve content, retrieve metada
...
++
+For the `List Item Query` operation the query should have this format: `LIST_ID?$select=FIELDS_TO_SELECT&filter=FILTER_PART`.
```
Comment (dhirsch1) [inline modules/ROOT/pages/sharepoint/sharepoint-connector.adoc]: Change to: For the `List Item Query` operation, the query has this format:

File: modules/ROOT/pages/sharepoint/sharepoint-connector.adoc [hunk 3]
```diff
@@ -196,6 +196,18 @@ For Files and Folders, operations include add, retrieve content, retrieve metada
...
++
+For example: `Vessels?$select=Title,FLAG,SEGMENT,ID,VesselName&$filter=Title eq 1`.
++
+The same format also applies to the other query operations.
```
Comment (dhirsch1) [inline modules/ROOT/pages/sharepoint/sharepoint-connector.adoc]: Remove "also" from the sentence.

File: modules/ROOT/pages/sharepoint/sharepoint-connector.adoc [hunk 4]
```diff
@@ -196,6 +196,18 @@ For Files and Folders, operations include add, retrieve content, retrieve metada
...
++
+You can learn more about building queries at the SharePoint documentation page https://docs.microsoft.com/en-us/sharepoint/dev/sp-add-ins/use-odata-query-operations-in-sharepoint-rest-requests[Use OData query operations in SharePoint REST requests].
```
Comment (dhirsch1) [inline modules/ROOT/pages/sharepoint/sharepoint-connector.adoc]: To learn more about building queries see the SharePoint documentation page...

**Review by dhirsch1 — CHANGES_REQUESTED** [review]
A few small changes to make

File: modules/ROOT/pages/sharepoint/sharepoint-connector.adoc (hunk 1 above)
Comment (alexandruatomei9) [reply modules/ROOT/pages/sharepoint/sharepoint-connector.adoc]: added

File: modules/ROOT/pages/sharepoint/sharepoint-connector.adoc (hunk 2 above)
Comment (alexandruatomei9) [reply modules/ROOT/pages/sharepoint/sharepoint-connector.adoc]: done

File: modules/ROOT/pages/sharepoint/sharepoint-connector.adoc (hunk 3 above)
Comment (alexandruatomei9) [reply modules/ROOT/pages/sharepoint/sharepoint-connector.adoc]: removed

File: modules/ROOT/pages/sharepoint/sharepoint-connector.adoc (hunk 4 above)
Comment (alexandruatomei9) [reply modules/ROOT/pages/sharepoint/sharepoint-connector.adoc]: changed

**Comment by mauro-dentoni** [comment]
@george-hoffer, Is it possible that you help us reviewing this RN today. Diane has provided some adjustments that Alex has already applied. If we can have this checked today, it would be great. I'm asking this since Diane will be off. Thanks!!

**Review by dhirsch1 — APPROVED** [review]

**Pull request merged**